.github/workflows/e2e-suite.yml

```diff
@@ -3,9 +3,18 @@ name: E2E Tests
 on:
   workflow_dispatch:
     inputs:
-      commit_sha:
-        description: 'The hash value of the commit.'
-        required: true
+      python-version:
```

Comment:
When you get a chance, could [this fix](https://github.com/linode/linode_api4-python/pull/473/files) also be applied here?

Reply:
Thanks for the reminder!